spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/GcpTraceProperties.java

```diff
@@ -86,6 +86,11 @@
 	 */
 	private int messageTimeout = 1;
 
+	/**
+	 * Timeout in seconds for server response. Ref: https://github.com/spring-cloud/spring-cloud-gcp/issues/1617.
```

Comment:
I'm not sure we should have any URLs in the property description.
```suggestion
	 * Timeout in seconds for server response when sending traces.
```